src/modules/app/app.module.ts

```diff
@@ -18,11 +18,16 @@ import { SeedService } from '../seed/seed.service';
       envFilePath: `.env.${process.env.NODE_ENV}`,
       isGlobal: true,
     }),
-    TypeOrmModule.forRoot({
-      type: 'postgres',
-      url: process.env.DATABASE_URL,
-      entities: [ApiKey, Wallet, Event],
-      synchronize: true, // set to false in production
+    TypeOrmModule.forRootAsync({
+      useFactory: async () => {
+        return {
+          type: 'postgres',
+          url: process.env.DATABASE_URL,
+          schema: process.env.DATABASE_SCHEMA || undefined,
+          entities: [ApiKey, Wallet, Event],
+          synchronize: true, // Disable in production
```

Comment:
Let's set this to false altogether and rely on migrations alone @yunchipang

Reply:
@Kpoke do i submit another PR to set this to false?